scripts/py_matter_idl/matter_idl/matter_idl_parser.py

```diff
@@ -504,15 +505,25 @@ def idl(self, items):
         clusters = []
         endpoints = []
 
+        global_bitmaps = []
+        global_enums = []
+        global_structs = []
+
         for item in items:
             if isinstance(item, Cluster):
                 clusters.append(item)
             elif isinstance(item, Endpoint):
                 endpoints.append(item)
+            elif isinstance(item, Enum):
```

Comment:
just to clarify - do you know these are global because the non-global items are somewhere else? In the cluster?

Reply:
Yes, the name of the method is the lark toke name. So when I get `idl` called, it means all the elements passed in are the list of things that were parsed at the top level (i.e. in lark we say `idl: (struct|enum|bitmap|cluster|endpoint)*` and all these are at a top level guaraneteed. The individual enum/bitmap/struct at the cluster level are passed to the `cluster` method.